src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -592,6 +592,14 @@ private function addService($id, $definition)
             $return[] = sprintf("@throws InactiveScopeException when the '%s' service is requested while the '%s' scope is not active", $id, $scope);
         }
 
+        if ($definition->isDeprecated()) {
+            if ($return && 0 === strpos($return[count($return) - 1], '@return')) {
+                $return[] = '';
+            }
+
+            $return[] = '@deprecated';
```

Comment:
The thing is, it is the same property that store the custom message or the generic one, I can't determine when the default message is used or not. I don't think it has any sense of having yet another property for "deprecated tag:" message (as it would be used only in the php dumper moreover), especially as the default one is, as you said it, generic and (I think) not appropriate for this tag.

Using a generic message "This service is deprecated", without bothering with the given template could be good too, but I don't see the point too. IRC, in PHPStorm (and the plugin too), if there is a `@deprecated` tag, it just says the function is deprecated. without messages indeed... But honestly, I don't really think it is THAT bothersome not having a message. Especially if this message doesn't bring me more information, as it is the case with the default message (Captain Obvious). And I can't really determine if the default message is being used, if the message used has more information or whatnot...

But I do agree we should forbid the `\r`, `\`n` or `*/` in the message though.